Sample_v3/Samples/SwiftUISimpleChat/ChannelListView.swift

```diff
@@ -203,11 +203,13 @@ struct ChannelListView: View {
     }
     
     private func chatView(for index: Int) -> ChatView {
-        ChatView(
-            channel: channelList.controller.client.channelController(
-                for: channelList.channels[index].cid
-            ).observableObject
+        let channelController = channelList.controller.client.channelController(
+            for: channelList.channels[index].cid
         )
+        
+        channelController.listOrdering = .bottomToTop
```

Comment:
IMO because `ChatView` expects the ordering to be in a particular form, it should be `ChatView` who should set the ordering.

Reply:
Tried setting it in onAppear of ChatView before synchronize, but it causes the list to visibly change the ordering